pages/en/publish/pwa-play-billing.md

```diff
@@ -0,0 +1,222 @@
+| `"unknownSubscriptionUpgradeDowngradePolicy"` | No set policy. This is not recommended.                                                                                                                                                                                                |
+
+Learn more about the different proration modes in the Google Play Billing Library [reference documentation](https://developer.android.com/reference/com/android/billingclient/api/BillingFlowParams.ProrationMode). Check out the Android developer docs for more on [subscription upgrade and downgrades](https://developer.android.com/google/play/billing/subscriptions#change) and [proration mode recommendations](https://developer.android.com/google/play/billing/subscriptions#proration-recommendations).
+
+The usage of these additional fields will look something like this:
+
+```js
+const paymentMethod = [{
+	supportedMethods: ‘https://play.google.com/billing’,
+	data: {
+		sku: item.itemId,
+		oldSku: oldPurchase.itemId,
+		purchaseToken: oldPurchase.purchaseToken,
+		prorationMode: 'immediateAndChargeProratedPrice',
+	}
+}];
+
+```
+
+Here, `item` is the `ItemDetails` of the new subscription the user is trying to upgrade or downgrade to, and `oldPurchase` is the `PurchaseDetails` of the user’s current subscription.
+
+## Acknowledge a purchase
+
+After a user purchases an item, you should grant them the proper entitlements (access to the item or content they’ve just purchased). Then, use the Digital Goods API to acknowledge the purchase. Acknowledging a purchase lets Google Play know that you’ve received and processed the purchase appropriately.
+
+!!! aside.message--note
+**Note:** If a purchase is not acknowledged within 72 hours of the purchase time, the payment is refunded to the user and the purchase is revoked. The purchase token will no longer be valid, so when you [query for existing purchases](#check-existing-purchases) the revoked purchase won’t be returned. This ensures that a user isn’t improperly charged in the event of a network error which causes them to not be granted the entitlement to their item.
+!!!
+
+The Digital Goods API requires a purchase to be acknowledged with a `PurchaseType` of either `"onetime"` or `"repeatable"`. This is something that you determine client-side and is not something you set when you create your [in-app products and subscriptions in the Play Console](/{{locale.code}}/publish/play-console-setup-for-billing). It is recommended that you keep track of each item’s purchase type in your back-end server. If you also fetch your list of item IDs from the back-end server, you can store the item ID and purchase type as a key-value database.
+
+| Play Console   | `PurchaseType` | Description                                                                                                                                                                                                 |
```

Comment:
```suggestion
| Play Console   | PurchaseType | Description                                                                                                                                                                                                 |
```

Table headers shouldn't be formatted like content

Reply:
done!